incubator/hnc/pkg/kubectl/create.go

```diff
@@ -36,9 +38,20 @@ var createCmd = &cobra.Command{
 		// another namespace, this will be caught by the admission controller.
 		updates := hcUpdates{requiredChildren: args}
 		updateHC(client, updates, parent)
+		// Create hns instances.
+		createHNS(client, parent, args)
```

Comment:
I'm not sure if it's a good idea to do these both in the same command (`k hns create`). Shouldn't we be able to test these independently? I'd prefer `k hns create` to stay as it is for now, and maybe add `k hns create2` to test the new version. Or is there a good reason to keep them together?

Reply:
SG. I created a new `create2` and updated the commit title and body with more details.